modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -711,13 +711,10 @@ def _to_parquet_check_support(kwargs):
             Whether parallel version of `to_parquet` is applicable.
         """
         path = kwargs["path"]
-        compression = kwargs["compression"]
         if not isinstance(path, str):
             return False
         if any((path.endswith(ext) for ext in [".gz", ".bz2", ".zip", ".xz"])):
             return False
```

Comment:
@mvashishtha I looked at the implementation of `to_parquet` function in pandas, the use of compressed file extensions in the path name does not affect the compression algorithms. I believe that this check can be removed. What do you think about doing it in this pull request?

Reply:
Thanks @mvashishtha!